### PR TITLE
Pni improvements 5571

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/5571

Review URL: https://foundation-s-pni-improv-cpweal.herokuapp.com/privacynotincluded

### CODE REVIEW

File: source/sass/buyers-guide/views/catalog.scss [hunk 1]
```diff
@@ -118,7 +123,8 @@ body.catalog {
       flex: 0 0 auto;
       display: inline-block;
       background: url(../_images/buyers-guide/filter/search.svg);
-      background-size: 100% 100%;
+      background-size: 22px 22px;
+      background-repeat: no-repeat;
```
Comment (Pomax) [inline source/sass/buyers-guide/views/catalog.scss:127]: 100% is cutting off the icon by a single pixel, so this is the simplest solution I can see.

**Review by kristinashu — CHANGES_REQUESTED** [review]
The icon looks good now! 

The cursor/click area seems good above, below, and on the left side of the icon but is it possible to make it clickable to the right of the icon as well?

**Comment by Pomax** [comment]
yep, added a bit more spacing to the right side on desktop mode.

**Review by kristinashu — APPROVED** [review]
Nice, thank you!

**Review by mmmavis — APPROVED** [review]
Lovely! Ready for merge!

(I've ticked off the checkboxes in the ticket as well)

**Pull request merged**